lib/redis.js

```diff
@@ -131,8 +131,10 @@ module.exports = CoreObject.extend({
   _deploySuccessMessage: function(revisionKey) {
     var success       = green('\nUpload successful!\n\n');
     var uploadMessage = white('Uploaded revision: ')+green(revisionKey);
+    var exampleCommand = white('To activate, execute: \n\n')+
+      green('ember deploy:activate -r ' + revisionKey + '-e ' + this.config.environment);
```

Comment:
@LevelbossMike this most likely won't work, since `this.config.environment` is a guessed variable, and most likely undefined.

What would be the best way to access the environment?